src/utils/py/daos_api.py

```diff
@@ -679,7 +679,8 @@ def __init__(self, context, container, obj, rank=None, iotype=1,
         ctypes.memset(ctypes.byref(self.iod.iod_kcsum), 0, 16)
 
         # epoch range still in IOD for some reason
-        self.epoch_range = EpochRange()
+	# Commenting epoch_range because it was creating issue DAOS-2028.
```

Comment:
How do we know when to delete or uncomment this?

Reply:
As per Wangdi, iod will be updated later, So we will have to remove the comment once it's updated.

![image](https://user-images.githubusercontent.com/25516462/53920834-4eb49000-4023-11e9-99ff-97662fbd1218.png)